### ISSUE TITLE
Could not pass event PlayerJoinEvent to Panilla v1.2.3

### ISSUE THREAD

**Issue by x1p** [Ruinscraft/Panilla#16]
Saw this error happen when a player joined due to a fallback (another server closed so the player came to this server)
```
> [22:25:49 INFO]: UUID of player CrashTestPuppet is 89ac1d77-32c3-4bb3-b597-b67dbca6700c
> [22:25:49 ERROR]: Could not pass event PlayerJoinEvent to Panilla v1.2.3
java.util.NoSuchElementException: packet_handler
at io.netty.channel.DefaultChannelPipeline.getContextOrDie(DefaultChannelPipeline.java:1119) ~[patched_1.13.2.jar:git-Paper-624]
at io.netty.channel.DefaultChannelPipeline.addBefore(DefaultChannelPipeline.java:260) ~[patched_1.13.2.jar:git-Paper-624]
at io.netty.channel.DefaultChannelPipeline.addBefore(DefaultChannelPipeline.java:249) ~[patched_1.13.2.jar:git-Paper-624]
at com.comphenix.protocol.injector.netty.PipelineProxy.addBefore(PipelineProxy.java:66) ~[?:?]
at com.comphenix.protocol.injector.netty.ChannelInjector$3$1.addBefore(ChannelInjector.java:272) ~[?:?]
at com.ruinscraft.panilla.v1_13_R2.io.PlayerInjector.register(PlayerInjector.java:48) ~[?:?]
at com.ruinscraft.panilla.plugin.JoinQuitListener.onJoin(JoinQuitListener.java:15) ~[?:?]
at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor305.execute(Unknown Source) ~[?:?]
at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:69) ~[patched_1.13.2.jar:git-Paper-624]
at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[patched_1.13.2.jar:git-Paper-624]
at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:66) ~[patched_1.13.2.jar:git-Paper-624]
at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:520) ~[patched_1.13.2.jar:git-Paper-624]
at net.minecraft.server.v1_13_R2.PlayerList.onPlayerJoin(PlayerList.java:369) ~[patched_1.13.2.jar:git-Paper-624]
at net.minecraft.server.v1_13_R2.PlayerList.a(PlayerList.java:180) ~[patched_1.13.2.jar:git-Paper-624]
at net.minecraft.server.v1_13_R2.LoginListener.b(LoginListener.java:169) ~[patched_1.13.2.jar:git-Paper-624]
at net.minecraft.server.v1_13_R2.LoginListener.tick(LoginListener.java:65) ~[patched_1.13.2.jar:git-Paper-624]
at net.minecraft.server.v1_13_R2.NetworkManager.a(NetworkManager.java:271) ~[patched_1.13.2.jar:git-Paper-624]
at net.minecraft.server.v1_13_R2.ServerConnection.c(ServerConnection.java:129) ~[patched_1.13.2.jar:git-Paper-624]
at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:1132) ~[patched_1.13.2.jar:git-Paper-624]
at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:439) ~[patched_1.13.2.jar:git-Paper-624]
at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:940) ~[patched_1.13.2.jar:git-Paper-624]
at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:837) ~[patched_1.13.2.jar:git-Paper-624]
at java.lang.Thread.run(Thread.java:748) [?:1.8.0_212]
```
The player the lost it's connection

Panilla v1.2.3
Paper 1.13.2 build 624

**Comment by dextonanderson** [comment]
This should be fairly harmless. I am assuming you mean the player:

Was moved to a fallback server due to the other server shutting down.
Immediately lost connection to the fallback server (basically they were only connected for a split second).

I will include a fix for this in the next release.

**Comment by dextonanderson** [comment]
This may be fixed in 1.3.1, not sure though

**Comment by dextonanderson** [comment]
If this happens again, please comment here. I think this should be fixed
